utils.py

```diff
@@ -42,7 +42,14 @@ def n(v):
         return 0L
     if v.endswith('B'):
         return long(v[:-1])
-    return long(float(v))
+    return dbus.UInt64(long(float(v)))
+
+def n32(v):
```

Comment:
This function is returning an unsigned 32 bit integer.  Perhaps it should be named u32?